contracts/modules/elrond-wasm-module-governance/src/governance_configurable.rs

```diff
@@ -0,0 +1,197 @@
+elrond_wasm::imports!();
+
+#[elrond_wasm_derive::module]
+pub trait GovernanceConfigurablePropertiesModule {
```

Comment:
I think this module can be merged with the Governance module. Anyone who would want to use the Governance would want to only import one thing

Reply:
That's true, but still, even if it takes a slightly more complicated setup (read: add two modules instead of one to the main SC), the source code is a lot cleaner to read.

Reply:
Would be nice if including the "main" module would automatically include the sub-modules as well, but unfortunately, that's not currently possible, so I went for this compromise.